.travis.yml

```diff
@@ -0,0 +1,10 @@
+ language: node_js
+ node_js:
```

Comment:
After a key, please insert a blank line. Otherwise, remove all the blank lines consistently throughout the file